lumen/ai/controls.py

```diff
@@ -198,8 +199,13 @@ def _generate_media_controls(self, event):
             self._upload_tabs.clear()
             self._media_controls.clear()
             for filename, file in self._file_input.value.items():
-                encoding = detect_file_encoding(file_obj=file)
-                file_obj = io.BytesIO(file.decode(encoding).encode("utf-8")) if isinstance(file, bytes) else io.StringIO(file)
+
+                if pathlib.Path(filename).suffix.lower() == ".csv":
```

Comment:
I think we should also add a default encoding in the detection, or raise an error.